src/RailWay/java/PageObjects/HomePage.java

```diff
@@ -1,9 +1,10 @@
 package PageObjects;
-import Common.Constant.CONSTANT;
 
-public class HomePage extends GeneralPage{
-   public void  open(){
-       CONSTANT.WEBDRIVER.navigate().to(CONSTANT.RailwayURL);
-   }
+import Common.Constant;
+
+public class HomePage extends GeneralPage {
+    public void open() {
+        Constant.WEBDRIVER.navigate().to(Constant.RAILWAY_URL);
```

Comment:
this method to navigate browser from a blank page to home page. Do not put it here

Reply:
i have updated for it